configs/example.ini

```diff
@@ -42,11 +42,12 @@ db = postgresql+psycopg2:///pajbot?options=-c%%20search_path%%3Dpajbot1_streamer
 ; See https://developers.google.com/safe-browsing/v4/get-started for how to get such an API Key
 ;safebrowsingapi = OWwcxRaHf820gei2PTouLnkUZbEWNo0EXD9cY_0
 
-; Optional section if you want to configure how user ranks are refreshed
-; 0 (default) = refresh every 5 minutes or so
+; Optional section if you want to configure how & delay of which user ranks are refreshed
+; 0 (default) = refresh every 5 minutes or so. If rank_refresh_delay is specified (in minutes), then it will take that value.
 ; 1 = refresh once on startup only
 ; 2 = never refresh (Be wary of setting this on a completely new bot)
 ;rank_refresh_mode = 0
+;rank_refresh_delay = 5
```

Comment:
Don't modify the comment for `rank_refresh_mode`, add your own comment above `rank_refresh_delay`